packages/console/src/containers/AppContent/components/Topbar/index.tsx

```diff
@@ -8,6 +8,8 @@ import { isCloud } from '@/consts/cloud';
 import EarlyBirdGift from '@/onboarding/components/EarlyBirdGift';
 import GetStartedProgress from '@/pages/GetStarted/components/GetStartedProgress';
 
+import Contact from '../Contact';
+import Document from '../Document';
```

Comment:
Just double check are those components being used anywhere else? Should they nest under the Topbar component folder? In order to keep the AC component structure clean.

Reply:
Double checked these components, and move them to Topbar folder since they are not used out-side the scope.